packages/react-relay/relay-hooks/useLazyLoadQueryNode.js

```diff
@@ -69,7 +67,7 @@ hook useLazyLoadQueryNode<TQuery: OperationType>({
       fetchObservable,
       fetchPolicy,
       renderPolicy,
-      {start: startFetch, complete: completeFetch, error: completeFetch},
+      undefined,
```

Comment:
This was for calling `.unsubscribe()` when the component unmounts. However since the unsubscribe is now a no-op there's no point for keeping this anymore